### PR TITLE
WIP: Prepare release 2.7.1

### PR DESCRIPTION
**Pull request by joelsmith**
### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

**Comment by joelsmith** [comment]
Still needs to be tested with OperatorHub

**Review by zroubalik — COMMENTED** [review]
Looking good, just a fea nits around UI descriptors. GitHub UI doesn't allow me to comment on the exact locations in the files, since there hasn't been changes in this part. So I referenced by line numbers. Hope that makes sense.

**Comment by zroubalik** [comment]
@joelsmith could you please rebase to get latest fixes in the GH workflows?

**Comment by zroubalik** [comment]
> @joelsmith could you please rebase to get latest fixes in the GH workflows?

please rebase once this is merged: https://github.com/kedacore/keda-olm-operator/pull/143

File: config/manifests/bases/keda.clusterserviceversion.yaml [hunk 1]
```diff
@@ -111,15 +111,15 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Cloud Provider
     certified: "false"
-    containerImage: ghcr.io/kedacore/keda-olm-operator:2.6.1
-    createdAt: "2022-02-10T00:00:00.000Z"
+    containerImage: ghcr.io/kedacore/keda-olm-operator:2.7.1
```
[the remark this answers is not in the document]
Comment (joelsmith) [reply config/manifests/bases/keda.clusterserviceversion.yaml:114]: I added it here:
https://github.com/kedacore/keda-olm-operator/pull/136/files#diff-fabe582069e7ecdb1e5177441c7a85a0b8924defb58f78a0b801bb64229e3c8aR350
and here:
https://github.com/kedacore/keda-olm-operator/pull/136/files#diff-e29d9f0fbb216fed69e4715208b9ea9c7e7ac008023e73ca77a105310ee76ea9R350
Is that what you had in mind?

File: config/manifests/bases/keda.clusterserviceversion.yaml [hunk 2]
```diff
@@ -111,15 +111,15 @@ metadata:
...
+    createdAt: "2022-05-11T00:00:00.000Z"
     description: Operator that provides KEDA, a Kubernetes-based event driver autoscaler
     operatorframework.io/suggested-namespace: keda
     operators.operatorframework.io/builder: operator-sdk-v1.1.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     repository: https://github.com/kedacore/keda-olm-operator
     support: KEDA Community
-  name: keda.v2.6.1
+  name: keda.v2.7.1
```
[the remark this answers is not in the document]
Comment (joelsmith) [reply config/manifests/bases/keda.clusterserviceversion.yaml:122]: I think I understand, but please recheck.

**Comment by joelsmith** [comment]
Okay, requested changes made (assuming I understood what needed to happen) and rebased.

**Review by zroubalik — APPROVED** [review]
LGTM thanks!

**Pull request merged**